backend/capellacollab/core/logging/__init__.py

```diff
@@ -112,21 +113,26 @@ class LogRequestsMiddleware(base.BaseHTTPMiddleware):
     async def dispatch(
         self, request: fastapi.Request, call_next: base.RequestResponseEndpoint
     ):
-        get_request_logger(request).debug("request started")
+        started_at = datetime.datetime.now(datetime.UTC)
+        if not is_health_check_route(request):
+            get_request_logger(request).debug("request started")
         response: fastapi.Response = await call_next(request)
-        get_request_logger(request).debug(
-            "request finished", extra={"status_code": response.status_code}
-        )
+        if not is_health_check_route(request):
```

Comment:
Couldn't you cache result of `is_health_check_route(request)` here, to prevent it from being called twice?

Reply:
Done